runtimeconfig/google/cloud/runtimeconfig/config.py

```diff
@@ -187,7 +187,7 @@ def get_variable(self, variable_name, client=None):
           >>> from google.cloud import runtimeconfig
           >>> client = runtimeconfig.Client()
           >>> config = client.get_config('my-config')
```

Comment:
#3895 points out that this line should be `config = client.config('my-config')` (there is no `get_config` method).